source/locales/es/LC_MESSAGES/carbon_edge.po

```diff
@@ -0,0 +1,865 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2021, The Natural Capital Project
+# This file is distributed under the same license as the InVEST package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
+#
+#, fuzzy
```

Comment:
Out of curiosity, what does the `, fuzzy` indicate?

Reply:
It's put there by the tool that generates the PO file. [This post](https://stackoverflow.com/questions/15653093/what-does-fuzzy-mean-at-the-top-of-a-xgettext-generated-pot-file) suggests it's to indicate that the header needs to be filled in. I think that's okay for now since we haven't filled in the files yet.